snc.sh

```diff
@@ -44,6 +44,7 @@ ${PODMAN} run --pull=always --rm -i quay.io/coreos/ignition-validate:release - <
 mkdir ${PWD}/${CRC_INSTALL_DIR}/tmp/
 ${PODMAN} run --pull=always --rm -v ${PWD}/${CRC_INSTALL_DIR}/tmp/:/data:Z -w /data quay.io/coreos/coreos-installer:release download -a ${ARCH} -s stable -p qemu -f qcow2.xz --decompress
 sudo mv ${CRC_INSTALL_DIR}/tmp/fedora-coreos-*-qemu.${ARCH}.qcow2 /var/lib/libvirt/images/fedora-coreos-qemu.${ARCH}.qcow2
+sudo mv -Z ${CRC_INSTALL_DIR}/fcos-config.ign /var/lib/libvirt/images/
 rmdir ${PWD}/${CRC_INSTALL_DIR}/tmp/
 
 sudo setfacl -m u:qemu:rx $HOME
```

Comment:
This `setfacl` call can be removed now i think

Reply:
Ah yes.